index.html

```diff
@@ -418,71 +417,21 @@
       </section>
       <section>
         <h3>
-          Push Service Discovery and API Use
+          Push API Use
         </h3>
         <p>
-          This specification does not define either specific normative methods via which webapps
-          can determine the applicable push service, or initiate push requests through that service
-          via a push service API. While these limitations add complexity for webapp developers,
-          they reflect the current reality of a diversity of push services, for which convergence
-          at least on an API level, is not expected to be a short-term possibility. While a
-          converged push service API is a worthwhile goal and may be addressed in a different
-          specification, the Push API defined here has been designed to enable the current variety
-          of push services to be used as appropriate for the webapp host device or software
-          platform. This section provides examples of how a webapp can be designed to work with a
-          variety of hypothetical push services as needed.
-        </p>
-        <p>
-          The Push API provides several interface parameters and attributes that can enable a
-          webapp to deliver and obtain the necessary push service specific data enabling use of
-          that particular push service. These parameters and attributes include:
-        </p>
-        <ul>
-          <li>The <code>subscriptionId</code> of a <code>PushSubscription</code>, as an opaque
-          string to the Push API, may be used by push services to provide meaningful data to the
-          <a>application server</a> through the webapp. For example, the
-          <code>subscriptionId</code> may be used as a push service API request parameter or
-          request body element.
-          </li>
-          <li>As a URI, the <code>endpoint</code> of a <code>PushSubscription</code> provides a
-          flexible means for push services to deliver webapp-specific subscription and/or push
-          service API parameters to the <a>application server</a>, through the webapp.
-          </li>
-        </ul>
-        <p>
-          Push services that are compatible with the Push API will be expected to clarify how these
-          data are used in their APIs, e.g. through their developer programs. Examples of
-          considerations include:
+          The fields included in the <code>PushSubscription</code> is all the information needed for
+          an <a>application server</a> to send a <a>push message</a>. Push services that are
+          compatible with the Push API will be expected to provide an <a>endpoint</a> that conforms
+          to <em>TODO: refer to IETF spec</em>.  These parameters and attributes include:
```

Comment:
Wait a little longer.  We need people to review and comment on the proposed protocol; and to express their view that this should be adopted.